cageseq.nf

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env nextflow
+/*
+========================================================================================
+                                    nf-core/cageseq
+========================================================================================
+nf-core/cageseq Analysis Pipeline.
+#### Homepage / Documentation
+https://github.com/nf-core/cageseq
+----------------------------------------------------------------------------------------
+*/
+
+
+
+/*
+ * SET UP CONFIGURATION VARIABLES
+ */
+
+// Check if genome exists in the config file
+if (params.genomes && params.genome && !params.genomes.containsKey(params.genome)) {
+    exit 1, "The provided genome '${params.genome}' is not available in the iGenomes file. Currently the available genomes are ${params.genomes.keySet().join(", ")}"
+}
+
+params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
+params.bowtie_index = params.genome ? params.genomes[ params.genome ].bowtie1 ?: false : false
+params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
+params.gtf = params.genome ? params.genomes[ params.genome ].gtf ?: false : false
+
+// Check mandatory parameters
+if (params.input)   { ch_input = file(params.input) }   else { exit 1, 'Input not specified!'}
+if (params.fasta)   { ch_fasta = file(params.fasta) }   else { exit 1, 'Genome fasta file not specified!'}
+if (params.gtf)     { ch_gtf = file(params.gtf) }       else { exit 1, "No GTF annotation specified!"}
+
+// Get rRNA databases
+// Default is set to bundled DB list in `assets/rrna-db-defaults.txt`
+ribo_database = file(params.ribo_database_manifest)
```

Comment:
This whole part doesn't have to be run if `remove_ribo_rna` is set. Put it in an if-clause.